internal/provider/provider.go

```diff
@@ -61,7 +61,7 @@ const (
 	MongodbGovCloudQAURL           = "https://cloud-qa.mongodbgov.com"
 	MongodbGovCloudDevURL          = "https://cloud-dev.mongodbgov.com"
 	ProviderConfigError            = "error in configuring the provider."
-	MissingAuthAttrError           = "either Atlas Programmatic API Keys or AWS Secrets Manager attributes must be set"
+	MissingAuthAttrError           = "either Atlas Programmatic API Keys, Service Accounts, or AWS Secrets Manager attributes must be set"
```

Comment:
```suggestion
	MissingAuthAttrError           = "either AWS Secrets Manager, Service Accounts, or Atlas Programmatic API Keys attributes must be set"
ProviderMetaUserAgentExtra     = "user_agent_extra"
```

nit: text in hierarchical order

Reply:
changed in [01513e7](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/3700/commits/01513e7499f704481a0858f4022741838f0b1560)